### PR TITLE
# Display line breaks for readonly textarea fields

### PR DESCRIPTION
**Pull request by suezhou**
https://ncr.plan.io/issues/27972

### CODE REVIEW

File: app/builders/ndr_ui/bootstrap/readonly.rb [hunk 1]
```diff
@@ -23,6 +23,16 @@ def #{selector}(method, options = {}, *rest)
           END
         end
 
+        [:text_area].each do |selector|
```
Comment (kenny-lee-1) [inline app/builders/ndr_ui/bootstrap/readonly.rb]: `%i[text_area]` ?

File: app/builders/ndr_ui/bootstrap/readonly.rb [hunk 2]
```diff
@@ -23,6 +23,16 @@ def #{selector}(method, options = {}, *rest)
...
+          class_eval <<-END, __FILE__, __LINE__ + 1
+            def #{selector}(method, options = {}, *rest)
+              return super unless readonly?
```
Comment (kenny-lee-1) [inline app/builders/ndr_ui/bootstrap/readonly.rb:39]: blank line after return to keep rubocop happy

**Review by kenny-lee-1 — DISMISSED** [review]

**Review by ollietulloch — APPROVED** [review]

**Review by ollietulloch — CHANGES_REQUESTED** [review]
Sorry, commentedon wrong PR!

**Comment by suezhou** [comment]
@ollietulloch I think you replied to the wrong PR

PS Matrix is blocking the merging if we can't get the status. This PR is just too old.

**Review by ollietulloch — APPROVED** [review]

**Comment by ollietulloch** [comment]
> @ollietulloch I think you replied to the wrong PR
> 
> PS Matrix is blocking the merging if we can't get the status. This PR is just too old.
Yep, wrong PR, sorry!

Can we remove this PR and create another @suezhou?

**Comment by suezhou** [comment]
code recommit in #52

**Pull request closed without merging**